easybuild/easyconfigs/p/Python/Python-3.8.2-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,296 @@
+name = 'Python'
+version = '3.8.2'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['e634a7a74776c2b89516b2e013dda1728c89c8149b9863b8cea21946daf9d561']
+
+dependencies = [
+    ('binutils', '2.34'),  # required for pip install that involves compilation
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.31.1'),
+    ('XZ', '5.2.5'),
+    ('GMP', '6.2.0'),  # required for pycrypto
+    ('libffi', '3.3'),  # required for cryptography
+    # OS dependency should be preferred if the os version is more recent then this version,
+    # it's nice to have an up to date openssl for security reasons
+    # Python 3.7 requires OpenSSL >= 1.0.2, uncomment line below if OS version is too old (e.g. CentOS 6)
+    # ('OpenSSL', '1.1.1e'),
+]
+
+osdependencies = [('openssl-devel', 'libssl-dev', 'libopenssl-devel')]
```

Comment:
also depends on unzip

```
== FAILED: Installation ended unsuccessfully (build directory: /dev/shm/Python/3.8.2-GCCcore-9.3.0): build failed (first 300 chars): cmd "unzip -qq /mnt/nfs/tank/sources/p/Python/extensions/setuptools-45.2.0.zip" exited with exit code 127 and output:
/bin/bash: unzip: command not found
```

Reply:
fixed in 7a16d59d7c

Reply:
`unzip` should be installed as a dependency of Easybuild itself, how are you unziping any of the sources installed with it otherwise? In the same way that `tar` or `bzip` are also needed.

Reply:
ok, I missed the occasionally required section at https://easybuild.readthedocs.io/en/latest/Installation.html#dependencies

Reply:
We actually have `UnZip` easyconfigs, so why not add it as a build dep here?

Reply:
why do it? it should already be available as a requirement for Easybuild to work. Would you add `unzip` and/or `tar` as build deps to all easyconfigs?